mongo/errors.go

```diff
@@ -207,7 +207,8 @@ func (e MongocryptdError) Unwrap() error {
 	return e.Wrapped
 }
 
-type labeledError interface {
+// LabeledError is an interface for errors with labels.
+type LabeledError interface {
```

Comment:
Does this need to be exported? I think "ServerError" can embed an unexpected interface with exported functions.